codegen/src/test/java/com/paypal/selion/plugins/DataReaderTest.java

```diff
@@ -29,6 +29,8 @@
 import com.paypal.selion.elements.HtmlSeLionElementList;
 
 public class DataReaderTest {
+    private final String YAML_V2 = "src/test/resources/SampleV2YamlPage.yaml";
+    private final String YAML_MOBILE = "src/test/resources/SampleMobilePage.yaml";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2360069956/issues/source?bid=2928425&fileBranchId=3219780#l33)